pkg/crc/machine/machine.go

```diff
@@ -181,11 +181,11 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error loading bundle metadata: %v", err)
 		}
 		if bundleName != filepath.Base(startConfig.BundlePath) {
-			logging.Fatalf("Bundle '%s' was requested, but loaded VM is using '%s'",
+			logging.Fatalf("Bundle '%s' was requested, but the loaded VM is using '%s'",
```

Comment:
not sure about `loaded`

Reply:
but an instance using '...' already exists?

Reply:
Replace loaded with existing?

Reply:
`existing` is what is used in code, so makes more sense

Reply:
Fixed.